centaurCwlRunner/src/main/scala/centaur/cwl/Outputs.scala

```diff
@@ -0,0 +1,104 @@
+package centaur.cwl
+
+import cats.effect.IO
+import centaur.api.CentaurCromwellClient
+import cromwell.api.model.SubmittedWorkflow
+import cwl.{CwlDecoder, CwlType, MyriadOutputType, File => CwlFile}
+import io.circe.Json
+import io.circe.generic.auto._
+import io.circe.literal._
+import io.circe.syntax._
+import shapeless.{Inl, Poly1}
+import spray.json.{JsNumber, JsObject, JsString, JsValue}
+
+import scalaz.syntax.std.map._
```

Comment:
why scalaz and not 🐱

Reply:
Map.intersectWith is :fire: : https://github.com/scalaz/scalaz/blob/series/7.3.x/core/src/main/scala/scalaz/syntax/std/MapOps.scala#L12

Not in cats :frowning: